src/test/run-make/thumb-none-qemu/script.sh

```diff
@@ -1,19 +1,20 @@
+#!/usr/bin/env bash
 set -exuo pipefail
 
 CRATE=example
 
 env | sort
-mkdir -p $WORK_DIR
-pushd $WORK_DIR
-    rm -rf $CRATE || echo OK
-    cp -a $HERE/example .
-    pushd $CRATE
+mkdir -p "$WORK_DIR"
+pushd "$WORK_DIR"
+    rm -rf "$CRATE" || echo OK
```

Comment:
pre-existing, but this looks _super_ sketchy. @sekineh do you remember why you added this? Did you mean to write `&&` instead?

Reply:
I meant `||` trying to catch the no existent path case.
But the entire `|| echo OK` seems not necessary because rm `-f` option already 'ignore nonexistent files and arguments' [[1]]

[1]: https://man7.org/linux/man-pages/man1/rm.1.html

Reply:
`test -d "$CRATE" && rm -rf "$CRATE` might be better.

Reply:
I think `rm -rf "$CRATE"` alone should be enough :) since like you said it ignores the files that are missing.